packages/wallet-ui/src/assets/types/assets.ts

```diff
@@ -2,19 +2,18 @@ import ethIcon from '../images/eth-icon.svg';
 import starknetIcon from '../images/starknet-icon.svg';
 import daiIcon from '../images/dai-icon.svg';
 import aaveIcon from '../images/aave-icon.svg';
+import usdcIcon from '../images/usdc-icon.svg';
+import usdtIcon from '../images/usdt-icon.svg';
 
-export const assetIcons = [ethIcon, daiIcon, aaveIcon];
-export enum AssetIconsIndex {
-  Ether = 0,
-  DAI = 1,
-  AAVE = 2,
-}
+export const assetIcons: Record<string, string> = {
+  Ether: ethIcon,
+  DAI: daiIcon,
+  AAVE: aaveIcon,
+  'USD Coin': usdcIcon,
+  'Tether USD': usdtIcon,
+  'Starknet Token': starknetIcon, // Default to starknetIcon for STRK
+};
 
 export const getAssetIcon = (assetName: string): string => {
-  let iconIndex = -1;
-  if (Object.keys(AssetIconsIndex).indexOf(assetName) !== -1) {
-    iconIndex = AssetIconsIndex[assetName as keyof typeof AssetIconsIndex];
-  }
-
-  return assetName ? assetIcons[iconIndex] : starknetIcon;
+  return assetIcons[assetName] || starknetIcon; // Use starknetIcon as fallback
```

Comment:
can we change to uses symbol, rather than name, i knew b4 was using name, which it was wrong